src/base/bittorrent/peeraddress.h

```diff
@@ -42,4 +42,15 @@ namespace BitTorrent
         static PeerAddress parse(const QString &address);
         QString toString() const;
     };
+
+    struct PeerEndpoint
```

Comment:
@thalieht
>A possible way is to create another struct and use it *only in* peerlistwidget.cpp

Here is incorrect place. Seems it should be private for PeerListWidget.